onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -288,19 +369,39 @@ onnxruntime::Status CreateOp(const OrtKernelInfo* info,
                                                ep->Type(),
                                                &kernel_create_info);
   ORT_RETURN_IF_ERROR(status);
-  onnxruntime::Node node;
+
+  std::vector<onnxruntime::NodeArg*> input_args;
+  for (int i = 0; i < input_count; ++i) {
+    auto arg_ptr = std::make_unique<onnxruntime::NodeArg>(std::to_string(i), nullptr);
+    input_args.push_back(arg_ptr.release());
+  }
+  std::vector<onnxruntime::NodeArg*> output_args;
+  for (int i = 0; i < output_count; ++i) {
+    auto arg_ptr = std::make_unique<onnxruntime::NodeArg>(std::to_string(i), nullptr);
+    output_args.push_back(arg_ptr.release());
+  }
+  auto tmp_node_holder = std::make_unique<onnxruntime::Node>(std::string("standalone_") + op_name, op_name, "", input_args, output_args, nullptr, domain);
+  NodePtr node_ptr(tmp_node_holder.release(), ReleaseNode);
   for (int i = 0; i < attr_count; ++i) {
     auto attr_proto = reinterpret_cast<const ONNX_NAMESPACE::AttributeProto*>(attr_values[i]);
-    node.AddAttributeProto(*attr_proto);
+    node_ptr->AddAttributeProto(*attr_proto);
   }
+
   auto kernel_def_builder = KernelDefBuilder::Create();
   kernel_def_builder->SetName(op_name);
   kernel_def_builder->SetDomain(domain);
   kernel_def_builder->SinceVersion(version);
-  OpKernelInfo instant_kernel_info(node, *kernel_def_builder->Build(), *ep, {}, {}, {});
+
+  static std::unordered_map<int, OrtValue> kEmptyValueMap;
+  static OrtValueNameIdxMap kEmptyNameMap;
+
+  OpKernelInfo tmp_kernel_info(*node_ptr.get(), *kernel_def_builder->Build(), *ep, kEmptyValueMap, kEmptyNameMap, kernel_info->GetDataTransferManager());
   std::unique_ptr<onnxruntime::OpKernel> op_kernel;
-  FuncManager func_mgr;
-  status = kernel_create_info->kernel_create_func(func_mgr, instant_kernel_info, op_kernel);
+
+  static FuncManager kFuncMgr;
+  status = kernel_create_info->kernel_create_func(kFuncMgr, tmp_kernel_info, op_kernel);
+  ORT_RETURN_IF_ERROR(status);
+  status = NodeRepo::AddNode(op_kernel.get(), node_ptr);
```

Comment:
Can create NodeRepo statically here and then call AddNode on that instance instance of making its individual members static. Also, pass node_ptr like move(node_ptr).